code/modules/jobs/job_types/detective.dm

```diff
@@ -14,6 +14,7 @@
 	exp_requirements = 180
 	exp_type = EXP_TYPE_CREW
 	exp_type_department = EXP_TYPE_SECURITY
+	special_notice = "You aren't a security officer, do not do their job for them. However, you can help them if they need immediate assistance."
```

Comment:
Same as the captain comment, also add a similar message for the warden as they shouldn't be out of the brig hunting suspects either.